tools/cli/test/e2e/cli/split_test.go

```diff
@@ -0,0 +1,36 @@
+package cli
+
+import (
+	"bytes"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSplit(t *testing.T) {
+	cliPath := NewBin(t)
+
+	t.Run("Split valid specs", func(t *testing.T) {
+		base := NewAtlasYAMLBaseSpecPath(t)
+
+		cmd := exec.Command(cliPath,
+			"split",
+			"-s",
+			base,
+			"-o",
+			"output.yaml",
+		)
+
+		var o, e bytes.Buffer
+		cmd.Stdout = &o
+		cmd.Stderr = &e
+		require.NoError(t, cmd.Run(), e.String())
+
+		ValidateVersionedSpec(t, NewValidAtlas20230101YAMLSpecPath(t), "output-2023-01-01.yaml")
```

Comment:
cool, should we add outout-2023 to git ignore or do we want to keep the file in the repo?

Reply:
We need them in the repo, the e2e tests is using them (loads the specs and compare them)

Reply:
I thought the output one should be generated by the test, no?

Reply:
yes, `output-2023-01-01.yaml` is the spec generated by the split command that run in the test which is validated against the `NewValidAtlas20230101YAMLSpecPath` which is the versioned spec downloaded from the vbranch and added to the `test/data/split` folder. To be clear, the test doesn't download the spec from the vbranch. I did it manually

Reply:
let me know if something is still missing, I will wait for this thread to be resolved before merging. Thanks